app/client/components/TypeConversion.ts

```diff
@@ -177,7 +177,8 @@ export async function prepTransformColInfo(docModel: DocModel, origCol: ColumnRe
           value = String(decodeObject(value)).trim();
           const tags: unknown[] = (value.startsWith('[') && gutil.safeJsonParse(value, null)) || value.split(",");
```

Comment:
Instead of removing quotes in line 181, it's better to replace `value.split(",")` with `csvDecodeRow(value)` (available in `app/common/csvFormat`). This will both strip quotes, and also handle commas in quoted values.